var/spack/repos/builtin/packages/py-scikit-learn/package.py

```diff
@@ -35,8 +35,9 @@ class PyScikitLearn(PythonPackage):
     version('0.15.2', 'd9822ad0238e17b382a3c756ea94fe0d')
     version('0.16.1', '363ddda501e3b6b61726aa40b8dbdb7e')
     version('0.17.1', 'a2f8b877e6d99b1ed737144f5a478dfc')
+    version('0.13.1', 'acba398e1d46274b8470f40d0926e6a4')
 
     depends_on('python@2.6:2.8,3.3:')
-    depends_on('py-setuptools',   type='build')
+    depends_on('py-setuptools',   type='build', when='@0.17.1:')
```

Comment:
Even in the latest release (0.18.1) there is still similar code to this, except `--single-version-externally-managed` is added to the list.

```
# Optional setuptools features
# We need to import setuptools early, if we want setuptools features,
# as it monkey-patches the 'setup' function
# For some commands, use setuptools
SETUPTOOLS_COMMANDS = set([
    'develop', 'release', 'bdist_egg', 'bdist_rpm',
    'bdist_wininst', 'install_egg_info', 'build_sphinx',
    'egg_info', 'easy_install', 'upload', 'bdist_wheel',
    '--single-version-externally-managed',
])
if SETUPTOOLS_COMMANDS.intersection(sys.argv):
    import setuptools

    extra_setuptools_args = dict(
        zip_safe=False,  # the package can run out of an .egg file
        include_package_data=True,
        extras_require={
            'alldeps': (
                'numpy >= {0}'.format(NUMPY_MIN_VERSION),
                'scipy >= {0}'.format(SCIPY_MIN_VERSION),
            ),
        },
    )
else:
    extra_setuptools_args = dict()
```

I did find the version that this option is added though, and it does appear to be the version directly after 0.13.1, 0.14.